src/data_template_generators.js

```diff
@@ -20,33 +20,24 @@ const generateDataTemplate = (courseData, pathLookup) => {
     course_image_caption_text: courseData["image_caption_text"]
       ? courseData["image_caption_text"]
       : "",
-    publishdate: courseData["first_published_to_production"]
-      ? moment(
-        courseData["first_published_to_production"],
-        INPUT_COURSE_DATE_FORMAT
-      ).format()
-      : "",
     instructors: {
       content: (courseData["instructors"] || []).map(instructor =>
         helpers.addDashesToUid(instructor["uid"])
       ),
       website: "ocw-www"
     },
-    departments:     helpers.getDepartments(courseData),
-    course_features: courseData["course_feature_tags"]
-      ? courseData["course_feature_tags"].map(courseFeature =>
-        helpers.getCourseFeatureObject(courseFeature, courseData, pathLookup)
+    department_numbers: helpers.getDepartmentNumbers(courseData),
+    course_features:    courseData["course_feature_tags"]
```

Comment:
Can we do `courseData["course_feature_tags"] || []` here to avoid the ternary operator?